pymc3/tests/test_distributions.py

```diff
@@ -1075,6 +1075,7 @@ def test_beta(self):
             {"alpha": Rplus, "beta": Rplus},
             lambda value, alpha, beta: sp.beta.logcdf(value, alpha, beta),
             n_samples=10,
+            decimal=5,
```

Comment:
This will probably lead to issues in float32 runs. Usually you have to do `decimal=select_by_precision(float64=x, float32=y)`. I had to do something similar for the logp test in my PR https://github.com/pymc-devs/pymc3/pull/4548/commits/59950dd75bea0a185634ced24768964168001a26

Reply:
Ah, all right, I'll change it to something like that.